tests/commands/test_make_variant.py

```diff
@@ -1,137 +1,132 @@
 from __future__ import annotations
 
-from pathlib import Path
+import itertools
 from typing import TYPE_CHECKING
 
 import pytest
 
 from tests.utils import assert_zips_equal
 from variantlib.commands.main import main
-from variantlib.validators.base import ValidationError
 
 if TYPE_CHECKING:
-    from pytest_mock import MockerFixture
+    from pathlib import Path
 
 
 @pytest.fixture
-def pyproject_toml(
-    tmp_path: Path,
-) -> Path:
-    pyproject_toml = tmp_path / "pyproject.toml"
-    pyproject_toml.write_text("""
-[variant.default-priorities]
-namespace = ["installable_plugin"]
-
-[variant.providers.installable_plugin]
-requires = [
-    "test-plugin-package @ file:///dev/null",
-]
-plugin-api = "test_plugin_package"
-""")
-    return pyproject_toml
-
-
-def test_make_null_variant(
-    pyproject_toml: Path,
-    tmp_path: Path,
-) -> None:
-    main(
-        [
-            "make-variant",
-            "-f",
-            "tests/artifacts/test_package-0-py3-none-any.whl",
-            "-o",
-            str(tmp_path),
-            "--pyproject-toml",
-            str(pyproject_toml),
-            "--null-variant",
-        ]
-    )
-    assert_zips_equal(
-        Path("tests/artifacts/test_package-0-py3-none-any-00000000.whl"),
-        tmp_path / "test_package-0-py3-none-any-00000000.whl",
-    )
+def non_variant_wheel(test_artifact_path: Path) -> Path:
+    whl_f = test_artifact_path / "test-package/dist/test_package-0-py3-none-any.whl"
+    if not whl_f.exists() or not whl_f.is_file():
+        raise FileNotFoundError(f"Test package wheel not found: `{whl_f}`")
+    return whl_f
 
 
 @pytest.fixture
-def mocked_plugin_installer(
-    mocker: MockerFixture,
-    test_plugin_package_req: str,
+def pyproject_toml(test_artifact_path: Path) -> Path:
+    pyproject_f = test_artifact_path / "test-package/pyproject.toml"
+    if not pyproject_f.exists() or not pyproject_f.is_file():
+        raise FileNotFoundError(
```

Comment:
I meant that you don't have to check for it at all, because it the file does not exist, the tested function will raise a clear exception.